packages/rest-typings/src/v1/chat.ts

```diff
@@ -885,7 +893,7 @@ export type ChatEndpoints = {
 	};
 	'/v1/chat.getThreadsList': {
 		GET: (params: ChatGetThreadsList) => {
-			threads: IMessage[];
+			threads: IThreadMainMessage[];
```

Comment:
Can we ensure that this type is indeed being returned by `chat.getThreadsList`? I see we're still returning `IMessage[]` there